bin/neurondemo.in

```diff
@@ -24,8 +24,7 @@ fi
 export NRNDEMO
 
 if [ ! -f "${NRNDEMO}neuron" ] ; then
-	(cd "${NRNDEMO}release" ; "${NRNBIN}nrnivmodl" ; cd .. ; rm -f neuron ; ln -s release/${MODSUBDIR}/special neuron)
+	(cd "${NRNDEMO}release" ; "${NRNBIN}nrnivmodl" ; cd .. ; rm -f neuron ; echo "#!/bin/sh\n\$(dirname \"\$0\")/release/${MODSUBDIR}/special \"\${@}\"" > neuron ; chmod +x neuron)
```

Comment:
Oh...this was another can of worms.
I am not testing this locally but if CI passes then it should be good IMO.